srcpkgs/s3backer/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 's3backer'
+pkgname=s3backer
+version=1.5.6
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="expat-devel fuse-devel libcurl-devel libressl-devel zlib-devel"
+short_desc="FUSE-based single file backing store via Amazon S3"
+maintainer="Samadi van Koten <samadi@vktec.org.uk>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/archiecobbs/s3backer"
+distfiles="https://s3.amazonaws.com/archie-public/s3backer/s3backer-$version.tar.gz"
+checksum=deea48205347b24d1298fa16bf3252d9348d0fe81dde9cb20f40071b8de60519
```

Comment:
Add the following to fix the *-musl builds:
if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
	makedepends+=" musl-legacy-compat"
fi